### PR TITLE
[G2M] Commit watch integration and mention in git conventions

### PR DESCRIPTION
**Pull request by woozyking**
https://github.com/EQWorks/common/blob/commit-watch/git.md#automated-enforcement

### CODE REVIEW

File: git.md [hunk 1]
```diff
@@ -186,19 +206,19 @@ e323208 Hub - deprecate `primary` for `primary_key`
...
+Based on this, combined with a good convention [on commits](#on-commits), we can leverage tools like (our own) [`release` CLI](https://github.com/EQWorks/release) to generate nicely arranged release notes material.
 
 ## On Pull/Merge Requests
 
 Try to maintain a good length and depth of the code changes within each request, make it digestible within 30 minutes or less.
 
-Try to keep a [`CHANGELOG`](https://keepachangelog.com)-like summaries of changes in the request description. Try to attach screenshots/videos to demonstrate the workflow and before/after differences. This can be automated/semi-automated through good practices [on commits](#on-commits).
+Try to keep a concise summaries of changes in the request description. Try to attach screenshots/videos to demonstrate the workflow and before/after differences. A good practices [on commits](#on-commits) helps tremendously on conveying the summary of changes, so rebase/squash/rephrase commits as needed to keep the commit history clean.
```
Comment (maluhoss) [inline git.md:215]: suggestions:
- should be 'Try to keep a concise `summary` of changes'
- should be 'A good practice'

File: git.md [hunk 2]
```diff
@@ -186,19 +206,19 @@ e323208 Hub - deprecate `primary` for `primary_key`
...
 
-Reference and mark relevant issues as fixed (`fixes #123`), or label the request itself when there are no relevant issues -- not both.
+Reference and mark relevant issues as fixed/closed (e.g.: `fixes #123`) when applicable.
 
-Start the request as a draft and/or mark in the title with `[WIP]` prefix when it needs more time to complete. Mark it ready to review, and/or with a `[G2M]` prefix in the title to signal its readiness.
+Start the request as a draft and/or mark in the title with a `[WIP]` prefix when it needs more time to complete. Mark it ready to review, and/or with a `[G2M]` prefix in the title to signal its readiness.
 
-When reviewer approved the PR, it's ready for merge. Three ways to meget it through GitHub:
+When reviewer approved the PR, it's ready for merge. There are three ways to meget it through GitHub:
```
Comment (maluhoss) [inline git.md:221]: suggestions:
- should be 'When a reviewer or reviewers approves the PR, it's ready for merging.'
- should be 'There are three ways to merge it through Github'

File: git.md [hunk 3]
```diff
@@ -211,17 +231,17 @@ The relevant commits are all rewritten to be brought on [top of the `HEAD` of th
 
 ## On Reviewing
 
-Try to set aside some quality time as a routine to proactively review others' pull/merge requests, as thoroughly as your bandwidth allows. If there's no time, and/or no willingness, do not just cast approval without reviewing.
+Try to set aside some quality time as a routine to proactively review others' pull/merge requests, as thoroughly as your bandwidth allows. If there's no time, and/or no willingness, do not just cast approve without reviewing. For GitHub repositories, this is the perfect time to make good use of [GitHub's notifcation with a `review-requested` filter](https://github.com/notifications?query=reason%3Areview-requested).
```
Comment (maluhoss) [inline git.md:234]: suggestions
- should be 'do not just approve without reviewing'
- typo in 'Github's notification'

File: git.md [hunk 4]
```diff
@@ -211,17 +231,17 @@ The relevant commits are all rewritten to be brought on [top of the `HEAD` of th
...
 
 When giving code change requests, think twice before you submit -- be reasonable and constructive, and be ready to defend your reasons.
 
 If the code change suggestion does not warrant definitive reasoning, but would still be constructive in your opinion, make it as a non-blocking comment.
 
-Where relevant, go through the changes in deploy preview or in your local environment to check for UI/UX flow as well as bugs.
+Where applicable, go through the changes in deploy preview or in your local environment to go through intended workflow, and check for potential issues.
 
-Prioritize on reviewing those that marked as bug fixes, then the new features. Use `G2M` as a search filter to minimize noise -- provided that it is a well-followed convention in the given project.
+Prioritize on reviewing those that marked as bug fixes, then the new features. Use `G2M` as a search filter to minimize noise -- provided that it is a well-followed convention in the given repository.
```
Comment (maluhoss) [inline git.md:244]: suggestion: should be 'reviewing those that are marked as bug fixes'

also what does 'it' refer to in the line 'provided that it is a well-followed convention in the given repository'? PRs?

Comment (woozyking) [reply git.md:244]: it - PR prefix. will include in the revision

**Review by maluhoss — APPROVED** [review]

**Pull request merged**